### PR TITLE
Support rhel9 imagestream

### PR DESCRIPTION
**Pull request by phracek**
This pull request adds support for testing imagestreams also on RHEL9

### CODE REVIEW

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
RHEL9 images stream test has passed
```bash
-----------------------------------------------
Running test test_php_imagestream (starting at 2022-10-14 09:43:32+02:00) ... 
-----------------------------------------------
Running image stream test for stream test/imagestreams/php-rhel.json and application https://github.com/sclorg/s2i-php-container.git with context test/test-app
Now using project "sclorg-test-2071" on server "https://api.core-serv-ocp.hosted.psi.rdu2.redhat.com:6443".

You can add applications to this project with the 'new-app' command. For example, try:

    oc new-app rails-postgresql-example

to build a new example application in Ruby. Or use kubectl to deploy a simple Kubernetes application:

    kubectl create deployment hello-node --image=k8s.gcr.io/e2e-test-images/agnhost:2.33 -- /agnhost serve-hostname

secret/6340056-core-services-apps-ci-pull-secret created
imagestream.image.openshift.io/php created
Creating project skipped.
[snipped]
Waiting for php-80-testing build pod to finish .............. DONE
Waiting for php-80-testing pod becoming ready .... DONE
Error from server (NotFound): deploymentconfigs.apps.openshift.io "php-80-testing" not found
  Checking APP using ct_os_test_response_internal 'http://172.30.94.53:8080' '200' 'Test PHP passed' ...
command POD not running yet, will start one called command-app registry.access.redhat.com/ubi7/ubi
pod/command-app created
Waiting for command POD .NAME                              READY   STATUS              RESTARTS   AGE
command-app                       0/1     ContainerCreating   0          1s
php-80-testing-1-build            0/1     Completed           0          48s
php-80-testing-78f645b5c6-gh8qv   1/1     Running             0          8s
.NAME                              READY   STATUS      RESTARTS   AGE
command-app                       1/1     Running     0          4s
php-80-testing-1-build            0/1     Completed   0          51s
php-80-testing-78f645b5c6-gh8qv   1/1     Running     0          11s
DONE
Error from server (NotFound): deploymentconfigs.apps.openshift.io "php-80-testing" not found
  Information about the image we work with:
Error from server (NotFound): deploymentconfigs.apps.openshift.io "php-80-testing" not found
error: resource name may not be empty
error: resource name may not be empty
  Check passed.
```

**Comment by phracek** [comment]
[test-all]

File: test/run-openshift-remote-cluster [hunk 1]
```diff
@@ -21,13 +21,13 @@ test_php_imagestream
 
 trap ct_os_cleanup EXIT SIGINT
 
-ct_os_check_compulsory_vars || exit $OC_ERR
```
Comment (zmiklank) [inline test/run-openshift-remote-cluster:24]: Why did you remove this?

Comment (phracek) [reply test/run-openshift-remote-cluster:24]: Upss, sorry it looks like typo. I will fix it. Thanks.

Comment (phracek) [reply test/run-openshift-remote-cluster]: Fixed

**Comment by phracek** [comment]
[test-all]

**Review by zmiklank — APPROVED** [review]
